### PR TITLE
Voice Signals Example Extension

### PR DESCRIPTION
**Pull request by luisheb**
## Describe the proposed changes
Added one cell to the Phonemes classification example showcasing the accuracy obtained with the model with optimized parameters.
## Checklist before requesting a review

- [X] I have performed a self-review of my code
- [X] The code conforms to the style used in this package
- [X] The code is fully documented and typed (type-checked with [Mypy](https://mypy-lang.org/))
- [X] I have added thorough tests for the new/changed functionality

### CODE REVIEW

**Comment by luisheb** [comment]
The Sphinx build is currently failing due to an `AttributeError` raised during the execution of several example scripts. Specifically, the `ModifiedBandDepth` object lacks the `_get_tags()` method, which is required by scikit-learn's pipeline system for compatibility checks. This affects three examples, `plot_depth_classification.py`, `plot_classification_methods.py`, and `plot_skfda_sklearn.py`, all of which use `ModifiedBandDepth` inside a scikit-learn pipeline wrapped in a `PerClassTransformer`. To resolve this, `ModifiedBandDepth` needs to implement the `_get_tags()` method or inherit from a base class that provides it, such as `sklearn.base.BaseEstimator`. Until then, these examples will continue to fail during documentation builds.

**Comment by vnmabus** [comment]
> The Sphinx build is currently failing due to an `AttributeError` raised during the execution of several example scripts. Specifically, the `ModifiedBandDepth` object lacks the `_get_tags()` method, which is required by scikit-learn's pipeline system for compatibility checks. This affects three examples, `plot_depth_classification.py`, `plot_classification_methods.py`, and `plot_skfda_sklearn.py`, all of which use `ModifiedBandDepth` inside a scikit-learn pipeline wrapped in a `PerClassTransformer`. To resolve this, `ModifiedBandDepth` needs to implement the `_get_tags()` method or inherit from a base class that provides it, such as `sklearn.base.BaseEstimator`. Until then, these examples will continue to fail during documentation builds.

Fixed in #695, please rebase or merge develop.

File: examples/datasets/plot_phonemes_classification.py [hunk 1]
```diff
@@ -183,6 +183,35 @@
 # score = accuracy_score(y_test, y_pred)
 # print(score)
 
+# %%
+# The optimal parameters obtained are `smoother__kernel_estimator__bandwidth`
+# = 0.001, `classifier__n_neighbors` = 37 and `classifier__metric__alpha` =
+# 0.01. We now train a new classifier with this parameters.
```
Comment (vnmabus) [inline examples/datasets/plot_phonemes_classification.py]:
```suggestion
# 0.01. We now train a new classifier with these parameters.
```

File: examples/datasets/plot_phonemes_classification.py [hunk 2]
```diff
@@ -183,6 +183,35 @@
 # score = accuracy_score(y_test, y_pred)
 # print(score)
 
+# %%
+# The optimal parameters obtained are `smoother__kernel_estimator__bandwidth`
+# = 0.001, `classifier__n_neighbors` = 37 and `classifier__metric__alpha` =
```
Comment (vnmabus) [inline examples/datasets/plot_phonemes_classification.py]: Put parameters' names in monospace.

```suggestion
# The optimal parameters obtained are ``smoother__kernel_estimator__bandwidth``
# = 0.001, ``classifier__n_neighbors`` = 37 and ``classifier__metric__alpha`` =
```

File: examples/datasets/plot_phonemes_classification.py [hunk 3]
```diff
@@ -183,6 +183,35 @@
...
+# 0.01. We now train a new classifier with this parameters.
+
+optimal_smoother = KernelSmoother(
+    NadarayaWatsonHatMatrix(
+        bandwidth=0.01,
```
Comment (vnmabus) [inline examples/datasets/plot_phonemes_classification.py:193]: You seem to have exchanged the bandwidth and alpha values, either here or in the description.

**Review by vnmabus — CHANGES_REQUESTED** [review]

File: examples/datasets/plot_phonemes_classification.py (hunk 3 above)
Comment (luisheb) [reply examples/datasets/plot_phonemes_classification.py:193]: You are right. Thanks!

**Review by vnmabus — APPROVED** [review]
